patches/src/main/kotlin/app/revanced/patches/instagram/misc/devmenu/EnableDevMenuPatch.kt

```diff
@@ -0,0 +1,31 @@
+package app.revanced.patches.instagram.misc.devmenu
+
+import app.revanced.patcher.patch.bytecodePatch
+import app.revanced.util.getReference
+import app.revanced.util.indexOfFirstInstructionReversedOrThrow
+import app.revanced.util.returnEarly
+import com.android.tools.smali.dexlib2.Opcode
+import com.android.tools.smali.dexlib2.iface.reference.MethodReference
+
+@Suppress("unused")
+val enableDevMenuPatch = bytecodePatch(
+    name = "Enable developer menu",
+    use = false
+) {
+    compatibleWith("com.instagram.android")
+
+    execute {
+        with(clearNotificationReceiverFingerprint.method) {
+            indexOfFirstInstructionReversedOrThrow {
+                val ref = getReference<MethodReference>()
```

Comment:
dont shorten variable names


```suggestion
                val reference = getReference<MethodReference>()
```